includes/definitions/discovery/imcopower-ls110.yaml

```diff
@@ -0,0 +1,69 @@
+mib: IMCO-LS110-MIB
+modules:
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: sdsTempSensValue
+                    num_oid: '.1.3.6.1.4.1.33283.1.30.5.1.3.{{ $index }}'
+                    index: 0
```

Comment:
Hi,
You need to define the index like "index: 'sdsTempSensValue.{{ $index }}'" and not specify a value.